### ISSUE TITLE
Node does not want to load the accessory

### ISSUE THREAD

**Issue by udochrist** [dinmammas/homebridge-robonect#1]
Hi,

I've installed using 
```
npm install -gs homebridge-robonect
```
and added the accessory. when starting the homebridge i get the following error:

```
2018-05-11 10:32:23,stdout,    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
2018-05-11 10:32:23,stdout,    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:26:16)
2018-05-11 10:32:23,stdout,    at new Server (/usr/local/lib/node_modules/homebridge/lib/server.js:56:24)
2018-05-11 10:32:23,stdout,    at Server._loadPlugins (/usr/local/lib/node_modules/homebridge/lib/server.js:138:22)
2018-05-11 10:32:23,stdout,    at Array.forEach (native)
2018-05-11 10:32:23,stdout,    at Server.<anonymous> (/usr/local/lib/node_modules/homebridge/lib/server.js:142:14)
2018-05-11 10:32:23,stdout,    at Plugin.load (/usr/local/lib/node_modules/homebridge/lib/plugin.js:72:22)
2018-05-11 10:32:23,stdout,    at require (internal/module.js:20:19)
2018-05-11 10:32:23,stdout,    at Module.require (module.js:498:17)
2018-05-11 10:32:23,stdout,    at Function.Module._load (module.js:439:3)
2018-05-11 10:32:23,stdout,    at tryModuleLoad (module.js:447:12)
2018-05-11 10:32:23,stdout,    at Module.load (module.js:488:32)
2018-05-11 10:32:23,stdout,    at Object.Module._extensions..js (module.js:580:10)
2018-05-11 10:32:23,stdout,    at Module._compile (module.js:571:32)
2018-05-11 10:32:23,stdout,    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-robonect/index.js:2:17)
2018-05-11 10:32:23,stdout,    at require (internal/module.js:20:19)
2018-05-11 10:32:23,stdout,    at Module.require (module.js:498:17)
2018-05-11 10:32:23,stdout,    at Function.Module._load (module.js:418:25)
2018-05-11 10:32:23,stdout,    at Function.Module._resolveFilename (module.js:470:15)
2018-05-11 10:32:23,stdout,Error: Cannot find module 'request'
```

shouldn't you list 'request' as a dependency in packages.json if you require it?

regards
udo

**Comment by dinmammas** [comment]
I sure should. I'll fix it.

**Comment by dinmammas** [comment]
Fixed in 1.0.14
